src/main/java/kr/pickple/back/common/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,150 @@
+package kr.pickple.back.common.exception;
+
+import static kr.pickple.back.common.exception.ExceptionCode.*;
+
+import java.time.format.DateTimeParseException;
+import java.util.List;
+import java.util.Objects;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.converter.HttpMessageNotReadableException;
+import org.springframework.validation.FieldError;
+import org.springframework.web.HttpRequestMethodNotSupportedException;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.servlet.NoHandlerFoundException;
+
+import kr.pickple.back.common.dto.ExceptionResponse;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@RestControllerAdvice
+public class GlobalExceptionHandler {
+
+    @ExceptionHandler(NoHandlerFoundException.class)
+    public ResponseEntity<ExceptionResponse> handleNoHandlerFoundException(final NoHandlerFoundException e) {
+        log.warn("{}, requestURL: {} ", COMMON_NOT_FOUND.getMessage(), e.getRequestURL(), e);
+
+        final ExceptionResponse exceptionResponse = ExceptionResponse.builder()
+                .code(COMMON_NOT_FOUND.getCode())
+                .message(COMMON_NOT_FOUND.getMessage())
+                .rejectedValues(new String[] {e.getRequestURL()})
+                .build();
+
+        return ResponseEntity.status(COMMON_NOT_FOUND.getStatus())
+                .body(exceptionResponse);
+    }
+
+    @ExceptionHandler(HttpRequestMethodNotSupportedException.class)
+    public ResponseEntity<ExceptionResponse> handleHttpRequestMethodNotSupportedException(
+            final HttpRequestMethodNotSupportedException e) {
+        log.warn(
+                "{}, supportedHttpMethods: {}, requestMethod: {}",
+                COMMON_METHOD_NOT_ALLOWED.getMessage(),
+                e.getSupportedHttpMethods(),
+                e.getMethod(),
+                e
+        );
+
+        final ExceptionResponse exceptionResponse = ExceptionResponse.builder()
+                .code(COMMON_METHOD_NOT_ALLOWED.getCode())
+                .message(COMMON_METHOD_NOT_ALLOWED.getMessage())
+                .build();
+
+        return ResponseEntity.status(COMMON_METHOD_NOT_ALLOWED.getStatus())
+                .body(exceptionResponse);
+    }
+
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    public ResponseEntity<ExceptionResponse> handleMethodArgumentNotValidException(
+            final MethodArgumentNotValidException e) {
```

Comment:
P1
- 매개변수를 내려서 작성할 때는 아래와 같이 하도록 컨벤션 맞춰주시면 좋을 것 같아요!
```suggestion
            final MethodArgumentNotValidException e
        ) {
```